server/src/databaseengine.cpp

```diff
@@ -25,40 +27,39 @@ void DatabaseEngine::connectToDatabase()
         qDebug() << "Database not found";
 }
 
-void DatabaseEngine::addNewUser(const QString &userName, const QString &login,
-                                const QString &password, const QString &email)
+void DatabaseEngine::registerUser(const User &newUser)
```

Comment:
а внутри метода есть старый пользователь? почему не просто user?
```suggestion
void DatabaseEngine::registerUser(const User &user)
```

Reply:
Исправил